src/main/shop/service/ParserImpl.java

```diff
@@ -0,0 +1,63 @@
+package service;
+
+import dto.Transaction;
+
+public class ParserImpl implements Parser {
+    private static final int TYPE_OPERATION = 0;
+    private static final int NAME_FRUIT = 1;
+    private static final int QUANTITY = 2;
+    private static final int COUNT_DATA = 3;
+    private static final String BALANCE = "b";
+    private static final String SUPPLY = "s";
+    private static final String PURCHASE = "p";
+    private static final String RETURN = "r";
+    private static final String SYMBOL_FOR_SPLIT = ",";
+
+    @Override
+    public Transaction parseLine(String line) {
+        int quantity = 0;
+        String nameFruit = "";
+        String typeOperation = "";
```

Comment:
why have you put these values here?
can we just declare these variables on lines 28-30 where we use them?